SampleApps/TestApp/ContentView.swift

```diff
@@ -312,9 +334,9 @@ struct MultipleIdentityView: View {
 
                 Button {
                     if extensions.isEdgeIdentityRegistered {
-                        MobileCore.unregisterExtension(AEPEdgeIdentity.Identity.self)
+                        MobileCore.unregisterExtension(AEPEdgeIdentity.Identity.self as! Extension.Type)
```

Comment:
why did we change this line?

Reply:
I saw errors and I had made this change to suppress it. Revisiting this and fixing this.